### ISSUE TITLE
Documentation Update: list of widgets

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3#6986]
Here's a list of outdated documentation files to keep track when updating. List based only on the images. Cannot think how to easily inspect text-based changes. Some have only minor changes. Many have only Report (or similar) moved to status bar, they aren't listed here. That said, it might make sense to update them at some point, just to have a full update every ... 10 years? :)

# Data
- [ ] File #6966
- [ ] Datasets #6967
- [ ] Data Table #6968
- [ ] SQL Table
- [ ] Paint Data #6977
- [ ] Data Info #6978
- [ ] Edit Domain
- [ ] Color (Save, Load, Reset buttons added) #6981 

# Transform
(resolved in https://github.com/biolab/orange3-doc-visual-programming/pull/7)
- [x] Data Sampler
- [x] Select Columns
- [x] Select Rows (minimal)
- [x] Transpose
- [x] Merge Data (minimal)
- [x] Select By Data Index
- [x] Aggregate Columns
- [x] Group by (some aggregations added and order changed)
- [x] Apply Domain
- [x] Impute
- [x] Continuize (wording different)
- [x] Randomize (minor)
- [x] Purge Domain (minor)
- [x] Formula (no change but VERY poor quality image)
- [x] Create Class
- [x] Python Script

# Visualize
(resolved in https://github.com/biolab/orange3-doc-visual-programming/pull/2)
- [x] Tree Viewer
- [x] Distribution
- [x] Scatter Plot
- [x] Line Plot (very minor)
- [x] Sieve Diagram (very minor)
- [x] Mosaic Display
- [x] FreeViz
- [x] RadViz
- [x] Heat Map (low-high to range)
- [x] Pythagorean Tree (very minor)
- [x] Nomogram
- [x] Scoring Sheet Viewer (should be redone after #6924 to have the same results each time)

# Model
- [x] Logistic Regression
- [x] AdaBoost
- [x] Neural Network
- [x] Stochastic Gradient Descent
- [x] Save Model

# Evaluate
- [x] Test and Score
- [x] Predictions
- [x] Confusion Matrix
- [ ] Performance Curve (minor)
- [x] Parameter Fitter (missing Example)

# Unsupervised
- [x] Distance File
- [x] Correlations
- [x] Distance Map (low-high to range)
- [x] Hierarchical Clustering
- [x] Manifold Learning (minor)
- [x] PCA (minor)
- [x] Neighbors (minor)
- [x] Distances
- [x] MDS
- [x] Save Distance Matrix

**Comment by borondics** [comment]
@ajdapretnar, I was discussing with @markotoplak and we argreed that I'll stop with this until there is a clear procedure how to do the screenshots and their labeling. Let me know when that is decided if I wouldn't see an update on it.

**Comment by ajdapretnar** [comment]
@borondics I think we have now converged to an agreement. I am doing standard retina screenshots. The stamper.py is updated with larger stamps. Everything is moved to https://github.com/biolab/orange3-doc-visual-programming. [Here](https://github.com/biolab/orange3-doc-visual-programming/pull/2) is a sample PR I did. Once it is merged, I think we can do everything in the same manner.

**Comment by ajdapretnar** [comment]
https://github.com/biolab/orange3-doc-visual-programming/pull/7
https://github.com/biolab/orange3-doc-visual-programming/pull/8
https://github.com/biolab/orange3-doc-visual-programming/pull/9
https://github.com/biolab/orange3-doc-visual-programming/pull/10

**Comment by ajdapretnar** [comment]
https://github.com/biolab/orange3-doc-visual-programming/pull/11
But Scoring Sheet doesn't show tooltip for me after the update.

**Comment by janezd** [comment]
@ajdapretnar , can we close this?

**Comment by ajdapretnar** [comment]
I wanted to, but the Data section is still not resolved. @borondics Are you willing to resubmit your changes or should I go ahead and do the Data as well?
